about.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="./assets/style/index.css">
+	<title>Tourism Website</title>
+</head>
+
+<body>
+	<header>
```

Comment:
- [ ] OPTIONAL
Your indents appear to have more than 2 spacing making code look wider apart. For a cleaner code let's ensure to use just 2 spaces for indenting. You can also set the said option in your document formatter plugin in your code editor settings in order automatically apply the appropriate spacing.